cuda_bindings/cuda/bindings/path_finder.py

```diff
@@ -1,11 +1,8 @@
 # Copyright 2024-2025 NVIDIA Corporation.  All rights reserved.
-#
 # SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
 
-from cuda.bindings._path_finder.load_nvidia_dynamic_library import (
-    load_nvidia_dynamic_library as _load_nvidia_dynamic_library,
-)
-from cuda.bindings._path_finder.supported_libs import SUPPORTED_LIBNAMES as _SUPPORTED_LIBNAMES
+from cuda.pathfinder import SUPPORTED_NVIDIA_LIBNAMES as _SUPPORTED_LIBNAMES
+from cuda.pathfinder import load_nvidia_dynamic_lib as _load_nvidia_dynamic_library
```

Comment:
Can we add some kind of deprecation warning if someone uses `cuda.bindings.path_finder` instead of `cuda.pathfinder`?

Reply:
I think we made the decision to keep this for now and introduce the warning in a future release.